incubating/aws-s3/step.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
   tags: []
   icon:
     type: svg
-    url: https://raw.githubusercontent.com/codefresh-io/steps/aws-s3/incubating/aws-cdk/images/s3-logo.svg
+    url: https://raw.githubusercontent.com/codefresh-io/steps/master/aws-s3/incubating/aws-cdk/images/s3-logo.svg
```

Comment:
this is still incorrect, still a reference to aws-cdk

Reply:
Updated URL